cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -965,6 +966,19 @@ pub(super) mod test {
         }
     }
 
+    #[test]
+    fn catalog_manifest_rejects_unknown_fields() {
+        let manifest = formatdoc! {"
+            {CATALOG_MANIFEST}
+
+            unknown = 'field'
+        "};
+
+        let result = toml_edit::de::from_str::<TypedManifest>(&manifest);
+
+        assert!(result.is_err(), "{:?}", result);
```

Comment:
Can we assert on the specific error to ensure that the test is catching the right behaviour, and not some other error, in the future?

Reply:
yeah probably, serde does have variants for that, i skewed to the "lazy" solution here

Reply:
@dcarley asserting the error message now

Reply:
Sweet 👍🏻